### ISSUE TITLE
[A11y] General - Parsing

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1887]
### Operating System

Windows

### Browser

Edge

### Affected page

Homepage

### Admin user?

Yes

### Issue Description

The new label around the various login radio buttons works in two of the cases, but for "Other Affiliation" the label is wrapping a div, which W3C says is invalid.

### Suggested Fix

I think this will need a label with the `for` attribute in order to keep the style working.

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

Run this through the parsing widget in Accessibility Insights or else check the DOM with a similar tool.  Verify there are no incorrectly nested elements.

**Comment by straleyb** [comment]
This should be fixed. The label is around the text inside the div now.
